gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Dataset.java

```diff
@@ -235,4 +233,42 @@ public Table create(String table, TableDefinition definition, BigQuery.TableOpti
   public BigQuery bigquery() {
     return bigquery;
   }
+
+  public static Builder builder(BigQuery bigquery, DatasetId datasetId) {
```

Comment:
We can remove `builder` methods in functional classes. We should remove them also in `resourcemanager`